src/main.rs

```diff
@@ -64,17 +157,24 @@ impl Styles {
 const MAX_USAGES_DISPLAYED: usize = 5;
 
 // Usage: cprintln!(Sytles::Data, <normal println args>)
```

Comment:
```suggestion
// Usage: cprintln!(Styles::Data, <normal println args>)
```
might as well fix it while we're here